### ISSUE TITLE
Add server killing functionality

### ISSUE THREAD

**Issue by jakestrahm** [joeyespo/grip#361]
The ability to close all current grip servers from within grip e.g. 
_grip --killall_

**Comment by anhtumai** [comment]
if you are on linux, you can use `pkill -f grip`

This article may help: https://stackoverflow.com/questions/8987037/how-to-kill-all-processes-with-a-given-partial-name

**Comment by HorridModz** [comment]
> if you are on linux, you can use pkill -f grip

Yes, but it would be nice if grip had built-in functionality that wrapped this.
